File: avalon-modules/avalon-authorization/src/main/java/io/github/lishangbu/avalon/authorization/repository/AuthenticationLogRepository.java
package io.github.lishangbu.avalon.authorization.repository;

import io.github.lishangbu.avalon.authorization.entity.AuthenticationLog;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AuthenticationLogRepository extends JpaRepository<AuthenticationLog, Long> {}

File: avalon-modules/avalon-authorization/src/main/kotlin/io/github/lishangbu/avalon/authorization/repository/AuthenticationLogRepository.kt
package io.github.lishangbu.avalon.authorization.repository

import io.github.lishangbu.avalon.authorization.entity.AuthenticationLog
import org.babyfish.jimmer.spring.repository.KRepository

/**
 * 认证日志仓储接口
 *
 * 定义认证日志数据的查询与持久化操作
 */
interface AuthenticationLogRepository : KRepository<AuthenticationLog, Long>
